docs/tutorial.rst

```diff
@@ -0,0 +1,1613 @@
+checking and other methods to correct model inconsistencies.
+
+In some cases there are reactions that are going to be inherently unbalanced
+and might cause problems with using these methods. If the user knows that
+this is the case for a specific reaction they can specify that the reaction
+be excluded from the masscheck so that the rest of the network can be
+analyzed. To do this the --exclude option can be used. For example if the
+user wanted to exclude the reaction FRUKIN from the masscheck they could
+use the following command:
+
+.. code-block:: shell
+
+    (psamm-env) $ psamm-model masscheck --exclude FRUKIN
+
+C) Formula Consistency Checking
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Formula checking will check that each reaction in the model is balanced
+with respect to the chemical formulas of each compound. To check the model
+for formula consistencies run the following command:
+
+.. code-block:: shell
+
+    (psamm-env) $ psamm-model formulacheck
+
+The output should appear as follows:
+
+.. code-block:: shell
+
+    INFO: Model: Ecoli_core_model
+    INFO: Model Git version: 70d186b
+    MANNIDEH	C27H40N7O20P2	C27H39N7O20P2		H
+    R_Biomass_Ecoli_core_w_GAM	C1088.0232H1471.1810N446.7617O1236.7018P240.5298S3.7478	C1045.4677H1395.2089N441.3089O1189.0281P236.8511S3.7478		C42.5555H75.9721N5.4528O47.6737P3.6787
+    INFO: Unbalanced reactions: 2/80
+    INFO: Unchecked reactions due to missing formula: 0/80
+
+In this case two reactions are identified in the model as being unbalanced.
+The biomass objective function, R_Biomass_Ecoli_core_w_GAM, and the
+reaction that was previously identified through masscheck as being
+unbalanced, MANNIDEH. In the case of the objective function this
+imbalanced due to the formulation of the objective function. The reaction
+functions as a sink for the compounds required for growth and only outputs
+depleted energy compounds. This leads to it being inherently formula
+imbalanced but it is a necessary feature of the model. The other reaction
+is MANNIDEH. It can be seen that the total number of atoms on each side
+does not match up. `PSAMM` also outputs what atoms would be needed to balance
+the reaction on both sides. In this case there is a missing hyrdrogen atom
+on the right side of the equation. This can be easily rectified by adding
+in the missing hydrogen. To do this correction in this tutorial the user
+can copy a fixed version of the mannitol pathway from the additional files
+folder using the following command:
+
+.. code-block:: shell
+
+    (psamm-env) $ cp ../additional_files/mannitol_pathway_v2.yaml ./mannitol_pathway.yaml
+
+Once that problem with the new reaction is fixed the model will pass both the
+formula check and masscheck.
+
+Now this fix can be added to the git repository so that the latest version
+of the model will be the fixed version. To do this the following commands
+can be used.
+
+.. code-block:: shell
+
+    (psamm-env) $ git add mannitol_pathway
```

Comment:
Should it be "(psamm-env) $ git add mannitol_pathway.yaml" instead of "(psamm-env) $ git add mannitol_pathway"